### DISCUSSION TITLE
Prune Propagation Policy available values description

### DISCUSSION THREAD

**Discussion by bygui86** [argoproj/argo-cd#6239] [Q&A] [answered]
Hi guys,

I cannot find anywhere a description about available values of Prune Propagation Policy:
- foreground
- background
- orphan

where can I find an explanation about them?

**Comment by chetan-rns** [comment, marked as answer]
Hi @bygui86, propagation policy is a concept in Kubernetes garbage collection. More details can be found here https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/. I suppose the same definition applies here for pruning as well

And by default, the prune propagation policy is `foreground` as mentioned here https://argoproj.github.io/argo-cd/user-guide/sync-options/#resources-prune-deletion-propagation-policy

**Reply by Vladyslav-Miletskyi** [reply]
this should be added to the docs!

**Reply by chetan-rns** [reply]
@Vladyslav-Miletskyi  I see you've raised a [PR](https://github.com/argoproj/argo-cd/pull/6555) for this. Thank you :)
